CNV consensus calls methods add (#93)
AlexsLemonade/OpenPBTA-manuscript: content/03.methods.md

CNVkit was run on human genome reference hg38 using the optional parameters of Theta2 purity and BAF adjustment for tumor-normal pairs.
We used GISTIC [@doi:10.1186/gb-2011-12-4-r41] v.2.0.23 on the CNVkit and the consensus CNV segmentation files to generate gene-level copy number abundance (Log R Ratio) as well as chromosomal arm copy number alterations using the parameters specified in the [OpenPBTA Analysis repository](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/run-gistic/scripts/run-gistic-openpbta.sh).

#### Consensus CNV Calling

For each caller and sample, CNVs were called based on consensus amongst ControlFreeC [@doi:10/ckt4vz; @doi:10/c6bcps], CNVKit [@doi:10.1371/journal.pcbi.1004873], and Manta [@doi:10/gf3ggb].

> CNVs called significant by ControlFreeC (pval = 0.01) were included in the consensus calling.

Reviewer: Should this be less than?

> Sample and caller combination files with more than 2500 CNVs called were removed from the set; we believe these to be noisy and poor quality samples (this is based on cutoffs used in GISTIC methods [@doi:10.1186/gb-2011-12-4-r41]).

Reviewer: ```suggestion
Sample and caller combination files with more than 2500 CNVs called were removed from the set; we expect these to be noisy and poor quality samples based on cutoffs used in GISTIC [@doi:10.1186/gb-2011-12-4-r41].
```
